MAUI/Segmented-Control/selection.md

```diff
@@ -0,0 +1,489 @@
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            TextColor = Colors.Red,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize Selected Segment Text Color of Each Segment Item
+
+You can change the selected segment text color of the each segment item by using `SelectedSegmentTextColor` property of `SfSegmentItem`.
+
+{% tabs %}
+{% highlight C# tabtitle="MainPage.xaml.cs"%}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        List<SfSegmentItem> itemsCollection = new List<SfSegmentItem>
+            {
+                new SfSegmentItem() {Text="Day", SelectedSegmentTextColor = Colors.LightBlue},
+                new SfSegmentItem() {Text="Week", SelectedSegmentTextColor = Colors.Blue},
+                new SfSegmentItem() {Text="Month", SelectedSegmentTextColor = Colors.SkyBlue},
+                new SfSegmentItem() {Text="Year", SelectedSegmentTextColor = Colors.DarkBlue},
+            };
+        segmentedControl.ItemsSource = itemsCollection;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Selected Segment Border Color
+You can customize the selected segment border color, by using `Stroke` property of `SelectionIndicatorSettings`.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage   
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.SelectionIndicatorSettings>
+            <buttons:SelectionIndicatorSettings 
+                Stroke="Green"/>
+        </buttons:SfSegmentedControl.SelectionIndicatorSettings>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SelectionIndicatorSettings = new SelectionIndicatorSettings()
+        {
+            Stroke = Colors.Green,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### Customize Selected Segment Border Thickness
+You can customize the selected segment border thickness, by using `StrokeThickness` property of `SelectionIndicatorSettings`.
```

Comment:
Customize the selected segment border thickness using the `StrokeThickness` property of `SelectionIndicatorSettings`.

Reply:
content changed